### PR TITLE
Add xmlWS-3.0 auto features

### PR DESCRIPTION
**Pull request by neuwerk**
This PR adds the auto features for xmlWS-3.0 and other Jakarta EE 9 features. It also includes changes to xmlBinding-3.0 to ensure that xmlWS-3.0 can start cleanly when included with other Jakarta EE 9 technologies.

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature [hunk 1]
```diff
@@ -23,6 +23,7 @@ Subsystem-Name: Jakarta EE Platform 9.0
  io.openliberty.messagingSecurity-3.0,\
  io.openliberty.messagingServer-3.0,\
  io.openliberty.webProfile-9.0,\
- io.openliberty.xmlBinding-3.0
+ io.openliberty.xmlBinding-3.0,\
+ io.openliberty.xmlWS-3.0
```
Comment (neuwerk) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature:27]: Not sure this should stay, but I added it to manual test that the feature played well with the other Jakarta EE features on server start-up.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.xmlwsSecurity-3.0
+visibility=private
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlWS-3.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0)))"
+-bundles=com.ibm.ws.jaxws.2.3.security.jakarta
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0:8]: This should be com.ibm.ws.jaxws.2.3.security.

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature (hunk 1 above)
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature:27]: Yeah you can't do this without marking the feature beta as well.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0 [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+IBM-Install-Policy: when-satisfied
+kind=noship
+edition=full
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0:11]: Can WLP-Activation-Type: parallel be added here as well?

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0 (hunk 2 above)
Comment (neuwerk) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0:8]: Nice catch, that was a bad merge on my part after the code reviews to the other PR.

File: dev/io.openliberty.xmlBinding.3.0.internal.tools/bnd.bnd [hunk 4]
```diff
@@ -53,9 +54,10 @@ Import-Package: \
   !org.apache.xerces*, \
   !org.jvnet*, \
   !com.sun.mirror.*, \
-  !com.sun.codemodel.*,\ 
+  com.sun.codemodel.*,\
   !com.sun.source.tree.*, \
   !com.sun.source.util.*, \
+  !com.sun.xml.dtdparser.*, \
```
Comment (neuwerk) [inline dev/io.openliberty.xmlBinding.3.0.internal.tools/bnd.bnd:60]: In wanting to make sure xmlWS-3.0 and xmlBinding-3.0 started correctly post-changes to auto feature files, I found two unresolved import errors for these two packages. The DTD Parser might need to be brought in to fully support xmlBinding-3.0 at some point, but at this point I'm not sure if it's required.

**Review by jhanders34 — APPROVED** [review]

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by jhanders34** [comment]
#libby

**Pull request merged**